qemu/tests/cfg/multi_disk.cfg

```diff
@@ -75,34 +86,50 @@
                     # We need to unload scsi_debug modules used by VM
                     kill_vm = yes
                     force_create_image = no
-                    pre_command = "modprobe -r scsi_debug; modprobe sg; modprobe scsi_debug add_host=9 dev_size_mb=40"
+                    pre_command = "sync; echo 3 > /proc/sys/vm/drop_caches; "
                     post_command = "rmmod scsi_debug"
                     stg_params += "image_raw_device:yes "
                     stg_params += "image_format:raw "
-                    stg_params += "indirect_image_select:range(-9,0) "
+                    Linux:
+                        # No need to remove partition for each disks since
+                        # partitions in all scsi_debug disks will be treated
+                        # as one
+                        rmpart_cmd = ''
+                        pre_command += "modprobe -r scsi_debug; modprobe sg; modprobe scsi_debug add_host=9 dev_size_mb=40"
+                        stg_params += "indirect_image_select:range(-9,0) "
+                    Windows:
+                        pre_command += "modprobe -r scsi_debug; modprobe sg; modprobe scsi_debug add_host=2 dev_size_mb=40"
+                        stg_params += "indirect_image_select:range(-2,0) "
```

Comment:
The same as above. Since the 'Linux:' part has its specific cmd 'rmpart_cmd = ''', so even move other cmds out of the part, it won't reduce the code quantity. Thus I just keep the original code layout which is more clear to understand.